synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, hs, media_repo, media_storage):
         )
 
     def render_OPTIONS(self, request):
+        request.setHeader(b"Allow", b"OPTIONS, GET")
```

Comment:
feels a bit odd to fix this here and not any of the other `render_OPTIONS` and `on_OPTIONS` tbh. Don't mind, though.